superset/tasks/schedules.py

```diff
@@ -33,24 +33,25 @@
 from flask import render_template, Response, session, url_for
 from flask_babel import gettext as __
 from flask_login import login_user
-from retry.api import retry_call
+from retry.api import retry, retry_call
```

Comment:
`schedules.py` is getting pretty big, would it make sense to have a`utils/slack.py`?